runtime/component/flora.js

```diff
@@ -43,5 +43,10 @@ Flora.prototype.remoteMethods = {
         logger.info('unexpected error on opening url', url, options, err.stack)
         res.end(0, [ JSON.stringify({ ok: false, message: err.message, stack: err.stack }) ])
       })
+  },
+  'yodart.decrypt-app-secret': function DecryptAppSecret (reqMsg, res) {
```

Comment:
How about `verify-app-secret`?

Reply:
Updated to aligned with `appLoader.verifyAndDecryptAppSecret`.